README.md

```diff
@@ -74,39 +44,174 @@ This program was created with the following tech stack and packages:
+
+#### Sample Rhubarb Output
+
+```JSON
+"mouthCues": [
+    { "start": 0.00, "end": 0.08, "value": "X" },
+    { "start": 0.08, "end": 0.23, "value": "B" },
+    { "start": 0.23, "end": 0.37, "value": "C" },
+    { "start": 0.37, "end": 0.44, "value": "G" },
+    { "start": 0.44, "end": 1.42, "value": "B" },
+    ... ]
+
+```
+
+Rhubarb uses the `recognizer` input to select the engine to use to during the process of creating a phoneme file. When
+```recognizer: english```,  Rhubarb will use the `pocketSphinx` engine to process the english audio. When
+```recognizer: phonetic```, Rhubarb will use the `Phonetic` recognizer instead. It has been observed that the `Phonetic`
+recognizer generates a response quicker, but provides a lower quality lip synchronization for the English language.
+
+Once the phoneme contents are created, they are sent to the next process.
+
+## The Animation Factory
+
+The animation factory uses the [phoneme output](#sample-rhubarb-output) from the phoneme factory to logically select frames
+from `speaking-portal-project/images` to generate the 2D animation. Each frame corresponds to exactly one phoneme,
+avatar, blink value, and body position. This frame data is compiled into a [frame data output](#frame-data-output) file.
+
+### Frame Data Output
+
+```JSON
+file ./images/character01/X_eyes2.png
+outpoint 0.08
+file ./images/character01/B_eyes0.png
+outpoint 0.20
+file ./images/character01/C_eyes0.png
+outpoint 0.07
+file ./images/character01/G_eyes0.png
+outpoint 0.07
+file ./images/character01/B_eyes0.png
+outpoint 0.98
+```
+
+The [frame data output](#frame-data-output) includes a path to the image and the duration of the frame for the animation
+video. This file is then sent to FFMPEG, a highly portable library capable of rendering image frames into an mp4
+output. Once the video file is created, it is then returned to the user as a response from the API.
+
+## Example
+
+Below is an example of the Barb avatar speaking, blinking, and changing poses.
+
+<img src = "docs/documentation/Images/AmberAnimated.gif" width="250px" height="300px">
 
 ## Limitations
 
-- The phoneme generation process controlled via Rhubarb can be quite computationally intensive, and is the current bottleneck for performance.
+- The phoneme generation process relies on Rhubarb, which can be quite computationally intensive. This is the current
+bottleneck for performance.
 - Blink frequency is chosen using a random probability
-- Avatar poses are chosen at random. A suggestion for future development would be to include sentiment analysis on the text and choose appropriate poses: however, this also depends on which poses are provided by the artist.
+- Avatar poses are chosen at random. A suggestion for future development would be to include sentiment analysis on the
+text and choose appropriate poses; however, this depends on the poses provided by the artist.
```

Comment:
```suggestion
text to choose appropriate poses.
```
Sentiment analysis isn't really dependent on the artist

Reply:
I didn't word that part correctly, but I'm trying to say something along the lines of the poses provided by the artist might not be related to a specific sentiment, so if sentiment or some other analysis is added, then the artist should be aware of what kind of poses need to be included. If that makes sense

Reply:
Ahhh. I see what you're saying - I think we could probably just say that that is sorta common sense though no?

Reply:
I think leaving it short and sweet with your suggestion is a good way to go